dm/pkg/checker/mysql_server_test.go

```diff
@@ -36,7 +37,10 @@ func TestMysqlVersion(t *testing.T) {
 		{"8.0.1-log", true},
 		{"8.0.20", true},
 		{"8.0.35", true},
-		{"8.1.0", false},
+		{"8.1.0", true}, // Innovation release
```

Comment:
Just a stupid, curious question, will we support older Innovation releases as well? I think it is fine...